web/pdf_thumbnail_viewer.js

```diff
@@ -13,25 +13,9 @@
  * limitations under the License.
  */
 
-'use strict';
-
-(function (root, factory) {
-  if (typeof define === 'function' && define.amd) {
-    define('pdfjs-web/pdf_thumbnail_viewer', ['exports',
-      'pdfjs-web/ui_utils', 'pdfjs-web/pdf_thumbnail_view'], factory);
-  } else if (typeof exports !== 'undefined') {
-    factory(exports, require('./ui_utils.js'),
-      require('./pdf_thumbnail_view.js'));
-  } else {
-    factory((root.pdfjsWebPDFThumbnailViewer = {}), root.pdfjsWebUIUtils,
-      root.pdfjsWebPDFThumbnailView);
-  }
-}(this, function (exports, uiUtils, pdfThumbnailView) {
-
-var watchScroll = uiUtils.watchScroll;
-var getVisibleElements = uiUtils.getVisibleElements;
-var scrollIntoView = uiUtils.scrollIntoView;
-var PDFThumbnailView = pdfThumbnailView.PDFThumbnailView;
+import { getVisibleElements, scrollIntoView, watchScroll
+  } from 'pdfjs-web/ui_utils';
```

Comment:
I went back and forth a bit on the matter of formatting multi-line `import` when writing the code.
In a way, I think it's more important that the `export`s are clear and readable, since ESLint will complain if you e.g. forget an import or import something that is unused.

However, if you think that readability is *too much* of an issue with the current `import` style, I can certainly change it as suggested.